cdap-kubernetes/src/main/java/io/cdap/cdap/master/environment/k8s/KubeMasterEnvironment.java

```diff
@@ -125,30 +119,6 @@ public void initialize(MasterEnvironmentContext context) throws IOException, Api
     discoveryService = new KubeDiscoveryService(namespace, "cdap-" + instanceName + "-", podLabels,
                                                 podInfo.getOwnerReferences());
 
-    // Optionally creates the pod killer task
-    String podKillerSelector = conf.getOrDefault(POD_KILLER_SELECTOR, DEFAULT_POD_KILLER_SELECTOR);
```

Comment:
Removed pod killer task as we no longer execute user code in preview manager pod.